src/data/nat/choose/sum.lean

```diff
@@ -103,6 +103,30 @@ begin
   simpa [sum_range_choose_halfway n] using t
 end
 
+lemma choose_middle_upper_bound (n : ℕ) : 4 ^ n ≤ (2 * n + 1) * choose (2 * n) n :=
```

Comment:
Should this be `pow_le_choose_middle`? (although I don't really know what "choose middle" means)

Reply:
"choose middle" in this file seems to mean a central binomial coefficient. It seems a little strange to me to use "pow" to describe the expression `4 ^ n` in these lemmas, but I guess that is how the other lemma does it.

Reply:
To be more specific, in the other lemma, choose middle is (2n+1 choose n) whereas here it is (2n choose n), so perhaps "central_binom" would be a better fragment.

Reply:
My suggestion is solely based on the name of the preceding lemma. All these terms ("middle," "central," "halfway") seem strange to me.

Reply:
Perhaps the most descriptive would be `four_pow_le_two_mul_add_one_mul_central_binom`, but this feels too wordy.
Perhaps the simplest would be `central_binom_upper_bound` - perhaps the rest of the file could be refactored. But what if someone comes along wanting to prove a tighter bound on the central binomial coefficient. I'm not sure what to do here

Reply:
Neither am I, let's wait for other opinions.

Reply:
Ok, I asked on Zulip, and concluded the long name is best, so I have pushed another change